node/repo/interface.go

```diff
@@ -69,6 +69,9 @@ type LockedRepo interface {
 	// SplitstorePath returns the path for the SplitStore
 	SplitstorePath() (string, error)
 
+	// SqlitePath returns the path for the Sqlite database
+	SqlitePath() (string, error)
```

Comment:
I think this is a bit ambiguous, since there's already a sqlite for events. Either we add that path here too, or we qualify this path to say that it's for the EthTxHash mapping.

Reply:
I do think we should map the events db to the same path. It gives an easy place to find the databases that we don't need to configure. 

It also allows us to keep the database inside the `LOTUS_PATH`, which I wasn't able to do with the setup from the events db (though that might have been my error with the way our dependency injection works).